rq_scheduler/scheduler.py

```diff
@@ -153,15 +153,16 @@ def schedule(self, scheduled_time, func, args=None, kwargs=None, interval=None,
                               job.id)
         return job
 
-
     def cron(self, cron_string, func, args=None, kwargs=None, repeat=None, queue_name=None):
         """
         Schedule a cronjob
         """
         scheduled_time = get_next_scheduled_time(cron_string)
 
+        # Set result_ttl to -1, as jobs scheduled via cron are periodic ones.
+        # Otherwise the job would expire after 500 sec.
         job = self._create_job(func, args=args, kwargs=kwargs, commit=False,
-                               result_ttl=None, queue_name=queue_name)
+                               result_ttl=-1, queue_name=queue_name)
```

Comment:
I think we should still allow the user to specify their own result_ttl. Could you please try to check for this in kwargs before defaulting to -1?